### DISCUSSION TITLE
Version issue

### DISCUSSION THREAD

**Discussion by Sure20220604** [timeseriesAI/tsai#535] [General]
Hi,
I want to know if version 0.3.2 has been released yet?
I didn't find it.
![fdd7d58960a54944587bf4318fb7c8a](https://user-images.githubusercontent.com/106857313/171983823-7bdefef4-95da-41f9-b368-9059ff60f049.png)
But author's file shows different:
![36347b0de1d8dcc575d77c358c47911](https://user-images.githubusercontent.com/106857313/171984100-d9bd8212-66ee-40a6-896c-a2762751180c.jpg)

**Comment by oguiza** [comment]
Hi @Sure20220604,
The reason for this apparent discrepancy is the following. The latest pip version is 0.3.1 (you can always check the latest version here https://pypi.org/project/tsai/). However, I sometimes work with the git version (when selecting stable = False). This pulls the code from the git repository, and it's always 1 version ahead of pip. When the git version's been tested we upload it to pip. And continue the cycle. So you may consider the git version like a work-in-progress version.

**Reply by Sure20220604** [reply]
Hi,@oguiza 
Okay,I got that.But  when I run "pip install git+https://github.com/timeseriesAI/tsai.git";there is always error and it's been bothering me for the whole afternoon.Thankfully I ran this code and it successfully got tsai 0.3.2 just now,I don't konw how but it's finally fixed.
Thank you again!
